src/test/resources/data/MultiGraphRequestCfgSample.json

```diff
@@ -42,6 +42,7 @@
       "minResultValue": 10,
       "maxResultValueRatio": 0.6,
       "maxResultValue": 1000,
+      "useProbabilityBasedWeightedAverage": true,
       "learntMaxEdgeWeight": 300,
       "filterValuesForSourceOrigins": ["POI:0"]
```

Comment:
don't forget the release note pls ;)

Reply:
do you mean the readme.md? I added a comment there already